### ISSUE TITLE
New backups created actually backup but do not show up in user interface.

### ISSUE THREAD

**Issue by jdcoats** [Cacti/plugin_routerconfigs#105]
```

2019/04/05 16:48:54 - CMDPHP SQL Backtrace:  
(/plugins/routerconfigs/router-download.php[133]:plugin_routerconfigs_download(),
/plugins/routerconfigs/include/functions.php[165]:plugin_routerconfigs_download_config(), 
/plugins/routerconfigs/include/functions.php[652]:db_execute_prepared())
2019/04/05 16:48:54 - CMDPHP ERROR: A DB Exec Failed!, Error: Unknown column 'lastuser' in 'field list'
2019/04/05 16:48:50 - CMDPHP SQL Backtrace:  
(/plugins/routerconfigs/router-download.php[133]:plugin_routerconfigs_download(), 
/plugins/routerconfigs/include/functions.php[165]:plugin_routerconfigs_download_config(),
/plugins/routerconfigs/include/functions.php[652]:db_execute_prepared())
2019/04/05 16:48:50 - CMDPHP ERROR: A DB Exec Failed!, Error: Unknown column 'lastuser' in 'field list'
```
![Screen Shot 04-05-19 at 04 51 PM](https://user-images.githubusercontent.com/42470924/55655832-358a3500-57c3-11e9-8cf4-079e40189a53.PNG)

Device | Filename
-- | --
TMH-NWRDC-02 | /var/www/localhost/htdocs/cacti/plugins/routerconfigs/backups/TMH-NWRDC-02-2019-04-05-1648

**Comment by netniV** [comment]
Can you check what version you are running and also describe your backup table.  Something seems wrong:

```sql
mysql> desc plugin_routerconfigs_backups;
+------------+--------------+------+-----+---------+----------------+
| Field      | Type         | Null | Key | Default | Extra          |
+------------+--------------+------+-----+---------+----------------+
| id         | int(11)      | NO   | PRI | NULL    | auto_increment |
| btime      | int(18)      | YES  | MUL | NULL    |                |
| device     | int(11)      | YES  | MUL | NULL    |                |
| directory  | varchar(255) | YES  | MUL | NULL    |                |
| filename   | varchar(255) | YES  |     | NULL    |                |
| lastchange | int(24)      | YES  | MUL | NULL    |                |
| lastuser   | varchar(64)  | YES  |     | NULL    |                |
+------------+--------------+------+-----+---------+----------------+
7 rows in set (0.00 sec)
```

**Comment by jdcoats** [comment]
It’s 1.5.0, I did update today also to check it. I also changed INFO to 1.4.0 and back up to 1.5.0 to see if a setup change got missed. I will  provide requested when I get back to a computer.


---
Sent from Workspace ONE Boxer<https://whatisworkspaceone.com/boxer>

**Comment by netniV** [comment]
You need to update the plugin_config table, not the info file to get a version update.

**Comment by jdcoats** [comment]
MariaDB [cacti]> desc plugin_routerconfigs_backups;
+------------+--------------+------+-----+---------+----------------+
| Field      | Type         | Null | Key | Default | Extra          |
+------------+--------------+------+-----+---------+----------------+
| id         | int(11)      | NO   | PRI | NULL    | auto_increment |
| btime      | int(18)      | YES  | MUL | NULL    |                |
| device     | int(11)      | YES  | MUL | NULL    |                |
| directory  | varchar(255) | YES  | MUL | NULL    |                |
| filename   | varchar(255) | YES  |     | NULL    |                |
| lastchange | int(24)      | YES  | MUL | NULL    |                |
| username   | varchar(64)  | YES  |     | NULL    |                |
+------------+--------------+------+-----+---------+----------------+
7 rows in set (0.001 sec)

**Comment by jdcoats** [comment]
No you don’t, you can change the info file and then go to the plugins page and see that the version has changed. If you check the table you can confirm that the version has changed also.


You need to update the plugin_config table, not the info file to get a version update.

**Comment by netniV** [comment]
Hmm, so it downgrades the version as well as upgrades it... never tested that ... the 'username' field should have become 'lastuser' and should now be corrected in 1.5.1

**Comment by jdcoats** [comment]
Thanks!
